client/src/pages/Register/InvitationInfo.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import { Alert } from 'react-bootstrap';
+import api from 'store/register/api';
+import Spinner from 'components/Spinner';
 
-import { type FormDataState } from './index';
+function InvitationInfo() {
+  const registrationApi = api.useGetRegistrationQuery();
+
+  if (
+    registrationApi.isFetching ||
+    registrationApi.isLoading ||
+    !registrationApi.data
```

Comment:
TODO: Check to see if any of these checks are redundant, and perhaps an error message if some of these states are possible and need some action to resolve.